src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3156,17 +3176,23 @@ def set_up_sku(self, mc: ManagedCluster) -> ManagedCluster:
         """
         self._ensure_mc(mc)
 
+        mc.sku = self.models.ManagedClusterSKU()
+        skuName = self.context.get_sku_name()
+        if skuName is not None and skuName == CONST_MANAGED_CLUSTER_SKU_NAME_AUTOMATIC:
+            mc.sku.name = "Automatic"
```

Comment:
I suppose `CONST_MANAGED_CLUSTER_SKU_NAME_AUTOMATIC.capitalize()` would be better

Reply:
The sku name that gets from Ln 3180 will be in lower case, why do we need to compare skuName using  `CONST_MANAGED_CLUSTER_SKU_NAME_AUTOMATIC.capitalize()` in Ln 3181?

Reply:
Discussed in the call. It's not related to the ln 3181.